library/platform_util.c

```diff
@@ -133,3 +134,14 @@ struct tm *mbedtls_platform_gmtime_r( const mbedtls_time_t *tt,
 #endif /* _WIN32 && !EFIX64 && !EFI32 */
 }
 #endif /* MBEDTLS_HAVE_TIME_DATE && MBEDTLS_PLATFORM_GMTIME_R_ALT */
+
+#if defined(MBEDTLS_CHECK_PARAMS) && defined(MBEDTLS_DEBUG_INVALID_PARAMS)
+void mbedtls_param_failed( const char* failure_condition, const char* file,
+                           int line )
+{
+#if defined(MBEDTLS_PLATFORM_C)
+    mbedtls_printf("%s:%i: Input param failed - %s\n", __FILE__, __LINE__,
+                   failure_condition );
+#endif
```

Comment:
Should something else happen here? It looks like with the default implementation of this function we will carry on executing with the failure instead of terminating or something like that.